crates/ruff_workspace/src/resolver.rs

```diff
@@ -212,8 +185,8 @@ fn resolve_configuration(
         // Resolve the current path.
         let options = pyproject::load_options(&path)
             .map_err(|err| anyhow!("Failed to parse `{}`: {}", path.display(), err))?;
-        let project_root = relativity.resolve(&path);
```

Comment:
I suspect that the regression comes from having moved the `project_root` out. Because it now longer gets resolved relative to the configuration (relevant when using extends)

Reply:
My understanding is that the project root is primarily used to resolve any relative paths.